README.md

```diff
@@ -14,7 +14,31 @@ An ES7/ES2016 spec-compliant `Array.prototype.includes` shim/polyfill/replacemen
 
 This package implements the [es-shim API](https://github.com/es-shims/api) interface. It works in an ES3-supported environment and complies with the proposed [spec](http://www.ecma-international.org/ecma-262/6.0/).
 
-Because `Array.prototype.includes` depends on a receiver (the “this” value), the main export takes the array to operate on as the first argument.
+Because `Array.prototype.includes` depends on a receiver (the `this` value), the main export takes the array to operate on as the first argument.
+
+## Getting started
+
+```sh
+npm install --save array-includes
+```
+
+or
+
+```sh
+yarn add array-includes
+```
+
+## Usage
+```js
+var includes = require('array-includes');
+var arr = [ 'one', 'two' ];
+
+includes(arr, 'one'); // true
+includes(arr, 'three'); // false
+includes(arr, 'one', 1); // false
+```
+
+`includes(array, value, [fromIndex=0])`
```

Comment:
this line is confusing - the "optional param" brackets a) usually include the preceding comma, and b) appear in documentation, not in code - but this makes it look like code. Can we tweak this?